server/chat-commands/info.ts

```diff
@@ -250,21 +248,16 @@ export const commands: ChatCommands = {
 			buf += `<br />Secret rooms: ${privaterooms}`;
 		}
 
-		const gameRooms = [];
-		for (const curRoom of Rooms.rooms.values()) {
-			if (!curRoom.game) continue;
-			const inPlayerTable = targetUser.id in curRoom.game.playerTable && !targetUser.inRooms.has(curRoom.roomid);
+		const gameRooms = targetUser.getRooms().filter(curRoom => {
```

Comment:
Actually wait this intentionally includes rooms the user isn't currently in! It actually only includes rooms the user isn't currently in! So this breaks that functionality.

Reply:
These changes should be mostly reverted.